### PR TITLE
docs: remove duplicate sentence from Routing

### PR DESCRIPTION
**Pull request by Mogglewump**
Sentence was followed by another with the same content.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/router.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 In a single-page app, you change what the user sees by showing or hiding portions of the display that correspond to particular components, rather than going out to the server to get a new page.
 As users perform application tasks, they need to move between the different [views](guide/glossary#view "Definition of view") that you have defined.
+
 To implement this kind of navigation within the single page of your app, you use the Angular **`Router`**.
```
Comment (kapunahelewong) [inline aio/content/guide/router.md:6]: While there's definitely duplication here, what about removing the sentence above it instead? That leaves the link to the glossary and specifically says what kind of navigation it is while fixing the duplicate content. What do you think?

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Thanks for this, @Mogglewump!  Just one small suggestion.

File: aio/content/guide/router.md (hunk 1 above)
Comment (kapunahelewong) [reply aio/content/guide/router.md:6]: For clarity, deleting this line instead:

> To implement this kind of navigation within the single page of your app, you use the Angular **`Router`**.

**Comment by atscott** [comment]
Hi @Mogglewump - are you still interested in contributing this docs improvement?

**Pull request closed without merging**